cloudinit/distros/freebsd.py

```diff
@@ -17,7 +17,13 @@
 LOG = logging.getLogger(__name__)
 
 
-class Distro(cloudinit.distros.bsd.BSD):
+class FreeBSD(cloudinit.distros.bsd.BSD):
```

Comment:
I realize `bsd.py` doesn't follow the trend of the class being named `Distro`, but can we leave that as the one exception and let this be named `Distro` here as well?